create.go

```diff
@@ -6,7 +6,7 @@ import (
 	"time"
 )
 
-// Create writes a new blank migration file.
+// Create : Writes a new blank migration file.
```

Comment:
I've never seen this convention before and I don't really like it.

What's wrong with standard formatting as in https://blog.golang.org/godoc-documenting-go-code ?